analysers/analyser_merge_parking_FR_BNLS.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Vincent Bergeot (thanks Frédéric Rodrigo) 2020             ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Parking_FR_BNLS(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        doc_main = dict(
+            detail = T_(
+'''It is not because it is from and OpenData source that it is good data. Review it before integrating. Your are an OSM contributors, not a machin to do blind import.'''),
+            fix = T_(
+'''If after review you are sure that it is a new data and right for OpenStreetMap, then you can add it.'''),
+            trap = T_(
+'''Be sure there is not existing it in an other place.'''))
+
+        doc_missing_official = self.merge_docs(doc_main,
+            detail = T_(
+'''This is issue if from an OpenData source, without any prior individual verification on this same issue.'''))
+        doc_possible_merge = self.merge_docs(doc_main,
+            detail = T_(
+'''This is a integration suggestion, mixing OpenData source and OpenStreetMap.'''))
+        doc_update_official = self.merge_docs(doc_main,
+            detail = T_(
+'''This is an update suggestion because there is the same ref in opendatabase and OSM.'''))
+
```

Comment:
Make this generic https://github.com/frodrigo/osmose-backend/commit/7144b6a43e0c6db9dd01be538836805802e57b80

```suggestion
```

Reply:
just to understand, when you say generic it's for all opendata ?

Reply:
yes